hbw/app/controllers/hbw/tasks_controller.rb

```diff
@@ -66,7 +66,7 @@ def task_id
     end
 
     def form_data
-      params[:form_data]
+      params[:form_data].to_h
```

Comment:
```to_h``` is not needed though. Check it, please.

Reply:
Checked. `ActionController::Parameters` object has no `map` method.